cluster/environments/azure-multiple-clusters-waf-tm-apimgmt/api-management-variables.tf

```diff
@@ -0,0 +1,33 @@
+variable "primary_region_waf_url" {
+  default = ""
+}
+
+variable "secondary_region_waf_url" {
+  default = ""
+}
+
+variable "tertiary_region_waf_url" {
+  default = ""
+}
+
+variable "traffic_manager_url" {
+  default = ""
+}
+
+variable "api_management_resource_group_name" {
+  default = "apimgmtresgrp5"
+}
+
+variable "region" {
+  default = "eastus"
+}
+
+variable "service_apim_name" {
+  default = "apimgmt-tm"
+}
+
+# variable "location" {
+#   description = "The location/region where the api management service will be deployed. The full list of Azure regions can be found at https://azure.microsoft.com/regions"
+#   type = "string"
+# }
+
```

Comment:
Why are these commented out? If not needed, they should be removed.

Reply:
@ReenuSaluja , take a look at the branch in your forked repo:
I have rebased it to upstream - microsoft/bedrock as well. 
https://github.com/ReenuSaluja/bedrock/tree/bedrockpull-sg/cluster/environments/azure-multiple-clusters-waf-tm-apimgmt

Reply:
Resolved and build test run successfully